daemon/manager/log/log.go

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2024 The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package logs
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"kmesh.net/kmesh/pkg/status"
+)
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "log",
+		Short: "Get or set kmesh-daemon's logger level",
+		Example: `Set default logger's level as "debug":
+		kmesh-daemon log --set default:debug
+	  
+	  Get default logger's level:
+		kmesh-daemon log default`,
+		Args: cobra.MaximumNArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			RunGetOrSetLoggerLevel(cmd, args)
+		},
+	}
+	cmd.Flags().String("set", "", "Set the logger level (e.g., default:debug)")
+	return cmd
+}
+
+func GetLoggerLevel(args []string) {
+	if len(args) != 1 {
+		fmt.Println("Missing logger name argument")
```

Comment:
nit: if we donot know what logger it is in kmesh, should we support kmesh log shows all the logger